ui/components/VerifiedStatus.tsx

```diff
@@ -23,17 +21,25 @@ const getVerifiedStatusColor = (status?: string) => {
 };
 
 export const findVerificationCondition = (
-  a: VerifiableSource
+  a: VerifiableSource | undefined
 ): Condition | undefined =>
   a?.conditions?.find((condition) => condition.type === "SourceVerified");
 
-export const VerifiedStatus = ({
+const checkVerifiable = (sourceRef: ObjectRef): boolean => {
+  const { name, namespace, kind } = sourceRef;
+  //can sourceRef actually return undefined stuff?! Typescript says it can.
```

Comment:
Yup just double checking in `objects.ts` and it looks like it's defined as ` sourceRef(): ObjectRef | undefined`